avm/res/web/serverfarm/main.bicep

```diff
@@ -136,7 +136,7 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableT
 
 resource appServicePlan 'Microsoft.Web/serverfarms@2022-09-01' = {
   name: name
-  kind: kind
+  kind: toLower(kind)
```

Comment:
Wonder if it would be better to just update the allowed set at the parameter :)

Reply:
In order to keep things backward compatible, this seemed like better option?

Reply:
As we're not yet in version 1.0 and because this is very very easy to adjust on the consumer side, I'd still vote to rather 'fix' the underlying issue. Thanks to the versioning we can do those changes 😉

Reply:
Ok then i will update accordingly and bump up the version

Reply:
@AlexanderSehr Have a look now!

Reply:
Looks great. Thanks for the thorough work.